public/views/partials/atoms/review-bar.liquid

```diff
@@ -0,0 +1,40 @@
+---
+metadata:
+  name: Review bar
+  params:
+    barClasses: ''
+    valueClasses: ''
+    maxValue: 120
+    value: 80
+    ariaLabel: 'myProgress'
+  styleguide:
+    - maxValue: 120
+      value: 120
+    - maxValue: 120
+      value: 70
+    - maxValue: 120
+      value: 20
+    - maxValue: 60
+      value: 20
+      barClasses: 'w-60 h-1 rounded bg-highlighted'
+      valueClasses: 'bg-colorful rounded h-full'
+---
+{% liquid
+  assign maxValue = params.maxValue | default: 100
+  assign value = params.value
+  assign ariaLabel = params.ariaLabel
+
+  assign percent = value | times: 100 | divided_by: maxValue
+
+  assign barClasses = params.barClasses | default :'w-52 h-1.5 border border-input-border bg-panel'
+  assign valueClasses = params.valueClasses | default :'bg-warning h-full'
+%}
+
+<div
+  role="progressbar"
```

Comment:
i think range would be better if this is not a "loading indicator"
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/range_role

Reply:
@nemvalid fixed